src/reducer/slices/status.ts

```diff
@@ -22,11 +21,12 @@ export interface Status {
     rejectValue: RejectValue;
     serialized: SerializedError;
   } | null;
+  contract?: String
```

Comment:
`String` as a type should be lowercase `string`

Reply:
```
Type 'String | undefined' is not assignable to type 'string | undefined'.
  Type 'String' is not assignable to type 'string'.
    'string' is a primitive, but 'String' is a wrapper object. Prefer using 'string' when possible.  TS2322
```

I get that when I try to use ```string```